### PR TITLE
[Enhancement](merge-on-write) parallel calculate delete bitmap when tablet has multi segments

### PR DESCRIPTION
**Pull request by liaoxin01**
## Proposed changes

Issue Number: close #xxx

In order to accelerate the processing time of publish for merge-on-write table，parallel calculate delete bitmap when tablet has multi segments.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by liaoxin01** [comment]
run buildall

**Comment by liaoxin01** [comment]
run buildall

**Comment by liaoxin01** [comment]
run buildall

File: be/src/service/point_query_executor.cpp [hunk 1]
```diff
@@ -264,7 +265,7 @@ Status PointQueryExecutor::_lookup_row_key() {
         // Get rowlocation and rowset, ctx._rowset_ptr will acquire wrap this ptr
         auto rowset_ptr = std::make_unique<RowsetSharedPtr>();
         st = (_tablet->lookup_row_key(_row_read_ctxs[i]._primary_key, true, nullptr, &location,
-                                      INT32_MAX /*rethink?*/, rowset_ptr.get()));
+                                      INT32_MAX /*rethink?*/, segment_caches, rowset_ptr.get()));
```
Comment (zhannngchen) [inline be/src/service/point_query_executor.cpp:268]: point lookup should not use such a segment cache? it's meaningless
you should allow to pass a nullptr here.

Comment (liaoxin01) [reply be/src/service/point_query_executor.cpp:268]: `_row_read_ctxs` may have multiple keys to lookup，so I think we also need a segment cache here.

**Comment by liaoxin01** [comment]
run buildall

**Review by zhannngchen — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**